lib/chef/provider/package/easy_install.rb

```diff
@@ -94,10 +93,10 @@ def load_current_resource
 
           if package_version == @new_resource.version
             Chef::Log.debug("#{@new_resource} at version #{@new_resource.version}")
-            @current_resource.version(@new_resource.version)
+            @current_resource.version(@new_resource.version) if @new_resource.version
```

Comment:
@lamont-granquist if the only purpose is to denote the presence or absence of a version, I'd argue for @current_resource.reset_property(:version) in the few cases where you want to *remove* a version (though I'm curious what those cases are).

I get the argument about guards for sure--I'm starting to like @dan's `NoContent` suggestion more and more for that reason.